src/nn/Model.js

```diff
@@ -167,6 +175,11 @@ export default class Model {
     if (!this._validateOperandList(outputs)) {
       throw new Error(`Invalid outputs ${outputs}`);
     }
+
+    if (type === OperationCode.BATCH_TO_SPACE_ND) {
```

Comment:
Please use a set to main the operators that tfjs wasm backend doesn't support and test against that.